client/sites/overview/components/hosting-overview.tsx

```diff
@@ -36,6 +38,11 @@ const HostingOverview: FC = () => {
 				title={ translate( 'Overview' ) }
 				subtitle={ subtitle }
 			/>
+			{ stickers.length > 0 && (
+				<div className="hosting-overview__blog-stickers">
+					{ translate( 'Blog Stickers: %s', { args: stickers.join( ', ' ) } ) }
+				</div>
+			) }
```

Comment:
Was this a test, or do we really intend to add the stickers in the overview?

![Captura de Tela 2025-05-16 às 15 15 33](https://github.com/user-attachments/assets/bec06918-5c2d-465b-ab4f-58e550c0fa77)

Reply:
That was a test that I forgot to remove. Thanks!